tests/test_sdk/test_connection.py

```diff
@@ -264,6 +316,72 @@ def test_two_epr():
     expected_text = """
 # NETQASM 0.0
 # APPID 0
+set R0 20
```

Comment:
Just a question (comment) about this test: Did you generate this instruction list manually upfront (what did you expect) or did you use the software and extracted the instruction list from a run. For the latter, the actual validation is then done before running this test (in checking if this instruction list is what you expected). This test is now only for finding regression in instruction generation which happens also with adding functionality. As we see here these tests have to be adjusted each time the instructions are somewhat different. Are you sure you want to keep these kind of tests (and future work)?

Reply:
Indeed the expected instruction list is generated up front, and these tests are only used to detect whether there are regressions (in later updates) or not. I believe this is still valuable, but I agree that updating these tests is a bit tedious. However I don't expect that these tests have to be updated much more often in the future (hopefully).